src-cpp/app/boot/app_controller.cpp

```diff
@@ -110,6 +115,8 @@ void AppController::connect()
     // self.statusFoundation.status.events.once("nodeStopped") do(a: Args):
     // TODO: remove this once accounts are not tracked in the AccountsModel
     // self.statusFoundation.status.reset()
+
+    // FIXME: use PointerToMember approach
```

Comment:
`SIGNAL` and `SLOT` macros approach is error-prone, its validity is not checked at compile time!

Reply:
@osmaczko I agree, we should always use functor based connections, but sometimes those macros are unavoidable.